packages/manager/apps/pci-savings-plan/src/types/CreatePlan.type.ts

```diff
@@ -25,3 +26,8 @@ export type Resource = {
   label: string;
   img: string;
 };
+
+export interface ListingProps {
```

Comment:
Props related to a React component should be scoped to the component itself, and if no need, should not be exported.